backend/tests/helpers/graphql.py

```diff
@@ -95,9 +96,13 @@ async def graphql_query(
     service: InfrahubServices | None = None,
     variables: dict[str, Any] | None = None,
     account_session: AccountSession | None = None,
+    refresh_schema: bool = False,
 ) -> ExecutionResult:
     branch = branch or await Branch.get_by_name(name="main", db=db)
     variables = variables or {}
+    if refresh_schema:
```

Comment:
please can you explain the use case behind the `refresh_schema` flag  ? in which situation do we need to force the schema to be refreshed ?

Reply:
I've removed this part.